navigation/Navigation.tsx

```diff
@@ -0,0 +1,9 @@
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
```

Comment:
we should be using the following tutorial:
https://reactnative.dev/docs/navigation